src/components/button/copy/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
 
-export const CopyButton = (props) => {
+export const CopyButton = (props: { copy: () => void; }) => {
```

Comment:
Here you can create an interface named CopyButtonProps and instead of import props import { copy } directly because is just one prop